students/srepking/Lesson09/donors.py

```diff
@@ -0,0 +1,140 @@
+
+
+class Group:
+
+    def __init__(self):
+
+        self._donor_raw = {}
+
+    def get_donors(self):
+        return self._donor_raw
+
+    donors = property(get_donors)
+
+    def search(self, donor):
+        if self._donor_raw.get(donor) is None:
+            return
+        else:
+            return donor
```

Comment:
can be simplified to:
```
def search(self, donor):
    return self._donor_raw.get(donor)
```
it's identical to your code as far as results.

Reply:
Yes! I fixed it and Thank you!